bin/juttle-engine-client

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env node
+
+/* eslint no-console: 0 */
+var _ = require('underscore');
+var minimist = require('minimist');
+var JuttleServiceClient = require('juttle-service/lib/juttle-service-client');
+var Promise = require('bluebird');
+var WebSocket = require('ws');
+var path = require('path');
+var cli_errors = require('juttle/lib/cli/errors');
+var open = require('open');
+var JuttleBundler = require('juttle-service/lib/bundler');
```

Comment:
A bit weary of reaching into `juttle-service` and grabbing these modules. My understanding is that not everything in `juttle-service-/lib` is _public_, but there is not distinction between what is and isn't. This means that when making changes to `juttle-service` like renaming or moving files, a developer will have no idea if its a breaking change and which semver version should be incremented.

Perhaps we can have one module within lib that exports everything that is public...or more everything that isn't public into some other directory?